terraform/gcp/variables.tf

```diff
@@ -1,40 +1,97 @@
-variable "user" {
-  description = "Instance user name that will used for SSH operations"
+variable "project" {
+  description = "Name of GCP project that will be used for housing require infrastructure"
   type        = string
 }
 
-variable "ssh_key" {
-  description = "Location on disk of the SSH public key to be used for instance SSH access"
+variable "name" {
+  description = "Name of provisioned resource"
   type        = string
-  default     = "~/.ssh/id_rsa.pub"
+}
+
+variable "instance_size" {
+  description = "Machine type to use for the instance"
+  type        = string
+  default     = "small"
 }
 
 variable "region" {
   description = "GCP region that'll be targeted for infrastructure deployment"
   type        = string
-  default     = "us-west1"
 }
 
 variable "node_count" {
-  description = "The quantity of nodes that are deployed within the environment for testing"
+  description = "Number of servers to deploy"
```

Comment:
I would say "Number of instances to provision"

Reply:
Sure.